api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -14,7 +14,7 @@ A user can click the **Export** button to save an Excel file with the exported d
 The following instructions show how to enable and configure client-side export:
 
 1. **Reference the required libraries**  
-    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries. Export also requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip</a> library. In a modular environment, this library is listed in package dependencies and is already added. If your app does not use modules, reference JSZip manually.
+    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries. If you use Angular, import **ExcelJS** v1.7.0 and add this library to the `tsconfig.app.json` file.
```

Comment:
Not only import but you also need to install the libraries. I suggest splitting the entire instructions by frameworks:

---
##### jQuery

Reference the libraries:

... code ...

##### Angular

Install the libraries:

> npm install ...

Import them:

... code ...

##### Vue 

... and so on ...